### PR TITLE
Updates to compose snapshot 6891475

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -80,15 +80,31 @@ dependencies {
...
-    implementation "androidx.compose.ui:ui:$rootProject.composeVersion"
-    implementation "androidx.compose.foundation:foundation:$rootProject.composeVersion"
-    implementation "androidx.compose.foundation:foundation-layout:$rootProject.composeVersion"
-    implementation "androidx.compose.material:material:$rootProject.composeVersion"
-    implementation "androidx.compose.ui:ui-viewbinding:$rootProject.composeVersion"
-    implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
-    implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
+    implementation("androidx.compose.runtime:runtime:$rootProject.composeVersion") {
+        version { strictly '1.0.0-SNAPSHOT' }
```
Comment (chrisbanes) [inline app/build.gradle:84]: You shouldn't need these changes with the proposed changes below

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 2]
```diff
@@ -275,7 +277,7 @@ private fun PlantImageHeader(
     Box(Modifier.fillMaxWidth()) {
         PlantImage(
             scrollState, imageUrl,
-            transitionModifier.onPositioned {
+            transitionModifier.onGloballyPositioned {
```
Comment (chrisbanes) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:280]: Use the new `onSizeChanged`

File: build.gradle [hunk 3]
```diff
@@ -70,12 +70,27 @@ plugins {
...
+        maven { // Accompanist snapshot repo
+            url "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
         google()
         jcenter()
         mavenCentral()
     }
 }
 
+configurations.all {
```
Comment (chrisbanes) [inline build.gradle:85]: You want this instead:

```
subprojects {
    configurations.configureEach {
        resolutionStrategy...
    }
}
```

**Review by tiembo — COMMENTED** [review]
LGTM pending reply to Chris's suggestions.  I'll make another review pass after that.

File: app/build.gradle (hunk 1 above)
Comment (manuelvicnt) [reply app/build.gradle]: Works! Thanks

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt (hunk 2 above)
Comment (manuelvicnt) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt]: Added, thanks!

File: build.gradle (hunk 3 above)
Comment (manuelvicnt) [reply build.gradle]: Yay! Thank you :)

**Review by chrisbanes — APPROVED** [review]

**Review by tiembo — APPROVED** [review]

**Review by tiembo — APPROVED** [review]

**Pull request merged**